config/config.md

```diff
@@ -321,15 +321,18 @@
-| `node_max_idle_time` | String | `24hours` | Max allowed idle time before removing node info from metasrv memory. |
-| `enable_telemetry` | Bool | `true` | Whether to enable greptimedb telemetry. Enabled by default. |
+| `store_tls` | -- | -- | TLS configuration for store backend (only applicable for PostgreSQL/MySQL backends)<br/>When using PostgreSQL or MySQL as metadata store, you can configure TLS here |
+| `store_tls.mode` | String | `disable` | TLS mode, refer to https://www.postgresql.org/docs/current/libpq-ssl.html<br/>- "disable" (default) - No TLS<br/>- "prefer" - Try TLS, fallback to plain<br/>- "require" - Require TLS<br/>- "verify_ca" - Require TLS and verify CA (not yet implemented)<br/>- "verify_full" - Require TLS and verify hostname (not yet implemented) |
+| `store_tls.watch` | Bool | `false` | Path to client certificate file (for client authentication)<br/>Path to client private key file (for client authentication)  <br/>Path to CA certificate file (for server certificate verification)<br/>Required when using custom CAs or self-signed certificates<br/>Leave empty to use system root certificates only<br/>Watch for certificate file changes and auto reload |
+| `store_tls.meta_table_name` | String | `greptime_metakv` | Table name in RDS to store metadata. Effect when using a RDS kvbackend.<br/>**Only used when backend is `postgres_store`.** |
+| `store_tls.meta_election_lock_id` | Integer | `1` | Advisory lock id in PostgreSQL for election. Effect when using PostgreSQL as kvbackend<br/>Only used when backend is `postgres_store`. |
+| `store_tls.selector` | String | `round_robin` | Datanode selector type.<br/>- `round_robin` (default value)<br/>- `lease_based`<br/>- `load_based`<br/>For details, please see "https://docs.greptime.com/developer-guide/metasrv/selector". |
+| `store_tls.use_memory_store` | Bool | `false` | Store data in memory. |
+| `store_tls.enable_region_failover` | Bool | `false` | Whether to enable region failover.<br/>This feature is only available on GreptimeDB running on cluster mode and<br/>- Using Remote WAL<br/>- Using shared storage (e.g., s3). |
+| `store_tls.region_failure_detector_initialization_delay` | String | `10m` | The delay before starting region failure detection.<br/>This delay helps prevent Metasrv from triggering unnecessary region failovers before all Datanodes are fully started.<br/>Especially useful when the cluster is not deployed with GreptimeDB Operator and maintenance mode is not enabled. |
+| `store_tls.allow_region_failover_on_local_wal` | Bool | `false` | Whether to allow region failover on local WAL.<br/>**This option is not recommended to be set to true, because it may lead to data loss during failover.** |
+| `store_tls.node_max_idle_time` | String | `24hours` | Max allowed idle time before removing node info from metasrv memory. |
+| `store_tls.enable_telemetry` | Bool | `true` | Whether to enable greptimedb telemetry. Enabled by default. |
```

Comment:
wait, this change might indicate we have a format issue with example toml. All the top level options are now under store_tls.

Reply:
And cert_path/key_path/ca_cert_path are not listed here

Reply:
oh my..